css/main.css

```diff
@@ -325,3 +325,90 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.footer {
+  display: flex;
+  flex-direction: column;
+}
+
+.footer .footer-header {
+  padding: 20px;
+  height: 216px;
+  margin: 90px 24px 80px;
+}
+
+.footer .footer-form {
+  margin: 20px 5px;
+}
+
+.footer .footer-header p {
+  text-align: center;
+  color: #172b4d;
+  font-size: 32px;
+  font-family: "Inter", sans-serif;
+  font-weight: bolder;
+  line-height: 42px;
+}
+
+.footer form {
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.footer .footer-form input {
+  margin: 10px 5px;
+  background: #fff;
+  box-sizing: border-box;
+  border-radius: 4px;
+  width: 90%;
+  height: 48px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  align-items: center;
+  color: #979493;
+  padding: 5px;
+  outline: none;
+  border: none;
+  border-bottom: 1px solid #d0d9d4;
+}
+
+.footer .footer-form textarea {
+  margin: 10px 5px;
+  background: #fbf8f7;
+  color: #3c3a39;
+  border: none;
+  outline: none;
+  border-radius: 4px;
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 400;
+  width: 65%;
+  box-sizing: border-box;
+  padding: 12px;
+  font-family: "Inter", sans-serif;
+  height: 114px;
+}
+
+.footer-form .see-project-button {
+  margin: 0;
+  color: white;
+  padding: 15px 10px;
+  border: none;
+  font-weight: bold;
+  background: #ff6b00;
+  height: 48px;
+  width: 40%;
+  font-size: 17px;
+  font-family: "Inter", sans-serif;
+  text-align: center;
+}
+
+.footer-form .see-project-button:hover {
+  color: #ff6b00;
+  background: #fff5e1;
+  border: 1px solid #f55800;
+}
```

Comment:
* [ ] You've done great here also but you're still missing some CSS-pseudo classes. 
- Add the `active`, `enabled`, and `disabled` pseudo-classes to your `.see-project-button` as this help to improve user experience.